hadoop-cloud/src/hadoop-3/test/scala/org/apache/spark/internal/io/cloud/CommitterBindingSuite.scala

```diff
@@ -125,22 +134,126 @@ class CommitterBindingSuite extends SparkFunSuite {
     val instance = FileCommitProtocol.instantiate(
       pathCommitProtocolClassname,
       jobId, "file:///tmp", false)
-
     val protocol = instance.asInstanceOf[PathOutputCommitProtocol]
     assert("file:///tmp" === protocol.destination)
   }
```

Comment:
If you don't mind, could you revert the newline addition on the above test case, `test("committer protocol can be serialized and deserialized")`?

Reply:
done. went and compared the history and reverted any newline changes which expanded the diff